### DISCUSSION TITLE
提交 SQL 检测失败

### DISCUSSION THREAD

**Discussion by phpandy3** [hhyo/Archery#2025] [General]
![image](https://user-images.githubusercontent.com/112546202/211441759-3e067b5a-fc75-4bcf-841e-ce7fe0d0222f.png)

请问下，为什么提交 SQL 一直提示这个错误，表结构和插入数据是没问题的。
实例是已经成功添加到后台中的，test 连接通过。

**Comment by lixingtao6234** [comment]
goinception没有启动成功，你可以看看这个启动的状态不对
